docs/contributing/Roslyn-Debugging.md

```diff
@@ -9,8 +9,14 @@ Sometimes it may be necessary to make changes in [`dotnet/roslyn`](https://githu
 3. Make the desired changes in `dotnet/roslyn`.
 4. `./Build.cmd -pack`. The `-pack` option causes the creation of NuGet packages.
 5. You should see the generated packages in the `roslyn\artifacts\packages\Debug\Release` directory. Take note of the package versions (ie. `Microsoft.CodeAnalysis.Workspaces.Common.3.8.0.nupkg` => `3.8.0`).
-6. Open `razor-tooling/NuGet.config` and add the local package source `<add key="Roslyn Local Package source" value="<PATH_TO_ROSLYN_REPO>\artifacts\packages\Debug\Release" />`.
-7. Open `razor-tooling/eng/Versions.props` and update all the `Tooling_*` versions to the version noted in step 5.
+6. Open `razor/NuGet.config` and add the local package source `<add key="Roslyn Local Package source" value="<PATH_TO_ROSLYN_REPO>\artifacts\packages\Debug\Release" />` and package source mapping
+   ```xml
+   <packageSource key="Roslyn Local Package source">
+     <package pattern="microsoft.*" />
+     <package pattern="microsoft.commonlanguageserverProtocol.*" />
```

Comment:
Surely the `microsoft.*` pattern covers this

Reply:
No, more specific patterns win: https://learn.microsoft.com/en-us/nuget/consume-packages/package-source-mapping#package-pattern-precedence

Reply:
Right, but isn't it still redundant having both for the same package source?

Reply:
I don't think so. I started with just `microsoft.*`, but then NuGet complained that `Microsoft.CommonLanguageServerProtocol.Framework` cannot be found, because it was trying to find it in different source. Adding that pattern is what worked for me.